living_documentation_regime/README.md

```diff
@@ -0,0 +1,358 @@
+    - Keep the documentation body free of technical solution specifics.
+    - Technical insights should be accessible through linked PRs or Tickets within the development repository.
+  - **Independent Documentation:** 
+    - Ensure the content remains independent of implementation details to allow a clear, high-level view of the feature or user story's purpose and functionality.
+
+### Labels
+
+To enhance clarity, the following label groups define and categorize each Documentation Issue:
+- **Topic**:
+  - **{Name}Topic:** Designates the main focus area or theme relevant to the ticket, assigned by the editor for consistency across related documentation.
+    - Examples: `ReportingTopic`, `UserManagementTopic`, `SecurityTopic`.
+  - **NoTopic:** Indicates that the ticket does not align with a specific topic, based on the editor's discretion.
+- **Type**:
+  - **DocumentedUserStory:** Describes a user-centric functionality or process, highlighting its purpose and value.
+    - Encompasses multiple features, capturing the broader goal from a user perspective.
+  - **DocumentedFeature:** Details a specific feature, providing a breakdown of its components and intended outcomes.
+    - Built from various requirements and can relate to multiple User Stories, offering an in-depth look at functionality.
+  - **DocumentedRequirement:** Outlines individual requirements or enhancements tied to the feature or user story.
+- **Issue States**:
+  - **Upcoming:** The feature, story, or requirement is planned but not yet implemented.
+  - **Implemented:** The feature or requirement has been completed and is in active use.
+  - **Deprecated:** The feature or requirement has been phased out or replaced and is no longer supported.
+
+**DocumentedUserStory** and **DocumentedFeature** serve as **Epics**, whereas **DocumentedRequirement** represents specific items similar to feature enhancements or individual requirements.
+
+### Hosting Documentation Tickets in a Solo Repository
+
+Using a dedicated repository solely for documentation tickets provides multiple advantages:
+- **Streamlined Management:** This avoids cross-project conflicts and board exclusions and enables specialized templates solely for documentation purposes.
+- **Focused Access Control:** This allows a small team to manage and edit documentation without interference, maintaining high-quality content.
+- **Optimized Data Mining:** Supports easier and more efficient data extraction for feedback and review cycles through Release Notes.
+- **Implementation Reflection:** Mirrors elements from the implementation repositories, providing a high-level knowledge source that is valuable for both business and technical teams.
+- **Release Notes Integration:** Documentation can evolve based on insights from release notes, serving as a dynamic feedback loop back to the documentation repository.
+
+---
+## Living Documentation Regime Features
+
+### Data Mining from GitHub Repositories
```

Comment:
Mention please that this feature is build-in and cannot be deactivate inside of regime scope.

Reply:
Mentioned at: [22593b3](https://github.com/AbsaOSS/living-doc-generator/pull/72/commits/22593b349c9ef8e9921c2ffeed07b527c8c2591b).